enterprise/settings.mdx

```diff
@@ -128,3 +128,23 @@ DEFAULT_ORG_ROLE=view_all
 ```
 
 Note you can run an [Enterprise self-check](/docs/enterprise/troubleshooting) to verify the connection and lookup LDAP information, but some configuration errors are only visible once logging in.
+
+## Sidekiq monitoring using Prometheus
+
+The metrics HTTP endpoint `/metrics` is available for the Prometheus scraping. Currently, it provides Sidekiq metrics using [sidekiq-prometheus-exporter gem](https://github.com/Strech/sidekiq-prometheus-exporter).
+
+The endpoint is protected with a bearer auth, the token is specified with the `SIDEKIQ_PROMETHEUS_TOKEN` environment variable. The endpoint is only available when the environment variable is set.
+If you have a Prometheus setup already, you can add the hostname and port of your web server as an additional target to the prometheus config file to scrape.
```

Comment:
For consistency, we should probably uppercase Prometheus in the later part of the sentence too:

```suggestion
If you have a Prometheus setup already, you can add the hostname and port of your web server as an additional target to the Prometheus config file to scrape.
```